gcloud/bigquery/table.py

```diff
@@ -693,6 +699,225 @@ def insert_data(self,
+
+        :type size: int
+        :param size: The number of bytes to read from the file handle.
+                     If not provided, we'll try to guess the size using
+                     :func:`os.fstat`. (If the file handle is not from the
+                     filesystem this won't be possible.)
+
+        :type num_retries: integer
+        :param num_retries: Number of upload retries. Defaults to 6.
+
+        :type allow_jagged_rows: boolean
+        :param allow_jagged_rows: job configuration option;  see
+                                  :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type allow_quoted_newlines: boolean
+        :param allow_quoted_newlines: job configuration option; see
+                                      :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type create_disposition: string
+        :param create_disposition: job configuration option; see
+                                   :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type encoding: string
+        :param encoding: job configuration option; see
+                         :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type field_delimiter: string
+        :param field_delimiter: job configuration option; see
+                                :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type ignore_unknown_values: boolean
+        :param ignore_unknown_values: job configuration option; see
+                                      :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type max_bad_records: integer
+        :param max_bad_records: job configuration option; see
+                                :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type quote_character: string
+        :param quote_character: job configuration option; see
+                                :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type skip_leading_rows: integer
+        :param skip_leading_rows: job configuration option; see
+                                  :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type write_disposition: string
+        :param write_disposition: job configuration option; see
+                                  :meth:`gcloud.bigquery.job.LoadJob`
+
+        :type client: :class:`gcloud.storage.client.Client` or ``NoneType``
+        :param client: Optional. The client to use.  If not passed, falls back
+                       to the ``client`` stored on the current dataset.
+
+        :rtype: :class:`gcloud.bigquery.jobs.LoadTableFromStorageJob`
+        :returns: the job instance used to load the data (e.g., for
+                  querying status)
+        :raises: :class:`ValueError` if size is not passed in and can not be
+                 determined
+        """
+        client = self._require_client(client)
+        connection = client.connection
+        content_type = 'application/octet-stream'
+
+        # Rewind the file if desired.
+        if rewind:
+            file_obj.seek(0, os.SEEK_SET)
```

Comment:
I just meant the `rewind` feature. (Sorry for being vague. Pronouns are hard to use.)